iteration.go

```diff
@@ -90,6 +90,48 @@ func (c *IterationController) Show(ctx *app.ShowIterationContext) error {
 	})
 }
 
+// Update runs the update action.
+func (c *IterationController) Update(ctx *app.UpdateIterationContext) error {
+	_, err := login.ContextIdentity(ctx)
```

Comment:
The check for login in the controller is a way way to avoid having to setup the whole security Middleware in the test. It's not used right now, but at some point the login will be used for all actions anyway